index.html

```diff
@@ -15,15 +15,19 @@
         <select id="selectedEpisode"
           >Select an episode</select
```

Comment:
if you want to show  "select an episode "  I think you need to add an option so your code will be like that
<select id="selectedEpisode" >
<option> select an episode</option></select>

Reply:
Although Buchra is correct, it's not necessary to have the 'Select an episode' option. I know you have a button that switches views, whereas others used the select box as the controller. It could be helpful to include the 'select n episode' option, however, so users know what the box is for :) a label could also be used